src/ibtracert.c

```
@@ -759,6 +761,13 @@ static int process_opt(void *context, int ch, char *optarg)
 	switch (ch) {
 	case 1:
 		node_name_map_file = strdup(optarg);
+		if (node_name_map_file == 0)
```

This additional check should probably be a separate patch and use NULL for the comparison.  Previous behavior just ignores the error.  Which I agree is probably not what we want.  I wonder what other tools ignore this error...  :-(